project/build.scala

```diff
@@ -0,0 +1,83 @@
+import sbt.Keys._
+import sbt._
+
```

Comment:
The problem is not with bindings but Cromwell-Backend. It's a separate project and a sub-project of Cromwell (in the future Cromwell-engine?)